### ISSUE TITLE
problem with docker.service issued from the latest node.yaml

### ISSUE THREAD

**Issue by rAy82** [GoogleCloudPlatform/kubernetes#3761]
Hello, I have a cluster of CoreOS machines built on VMware Workstation. I want to report a problem in starting the docker service using the recently modified node.yaml file.
It seems that my platform is having problems with the "overlay" option that was recently introduced in place of "brtfs", contained in the docker service definition:

ExecStart=/usr/bin/docker -d --bip=${FLANNEL_SUBNET} --mtu=${FLANNEL_MTU} -s=overlay -H fd://

The log states "driver not supported":

![ura](https://cloud.githubusercontent.com/assets/8778205/5879688/a4c5c01a-a331-11e4-8cf5-83d64f5e90c1.PNG)

Reverting that option to "brtfs" (and naturally rebooting), the docker service returns to work normally.
CoreOS version: 522.5.0
Docker version: 1.3.3
What do you think about it?

**Comment by dchen1107** [comment]
cc/ @kelseyhightower

**Comment by rAy82** [comment]
I have to correct my post: in one of the subsequent reboots I noticed the same error even with the "btrfs" option. It seems an occasional event whose cause I couldn't identify yet

**Comment by erictune** [comment]
@pires any idea why this is happening?  Is @rAy82 possibly using a different version of docker than you used to test your PR?

**Comment by pires** [comment]
@erictune yes, I've used Docker 1.4.1 which is part of the CoreOS alpha version that my PR deploys. Docker version bundle in each CoreOS version can be checked at.

@rAy82 you're using a stable version of CoreOS, aren't you? Can you change ````node.yaml``` and replace ```-s overlay``` with ```-s btrfs``` and give it a try? Let me know of the result so I open an issue and make this clear in the docs.

**Comment by rAy82** [comment]
Exacty, I'm using the latest stable version.
After a rebuilding a clean cluster I set the -s option from the docker unit to the "btrfs" value, rebooted and I found no problem wih the docker process.
To have a stronger confirmation of that, I switched again to overlay, rebooted and verified that the docker process was failing and then back again to btrfs, confirming that docker was ok.
When the new version of docker will be inclueded in a coreos stable release I must remember to modify the option to overlay again :-)
Thank you for your help
